client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -92,3 +106,17 @@ function PaymentInfoBlockWrapper( {
 		</aside>
 	);
 }
+
+function BackupPaymentMethodNotice() {
+	const translate = useTranslate();
+	/* translators: Notice that this purchase may use a backup payment method if it fails to renew */
+	const noticeText = translate( 'May use backup' );
```

Comment:
This wording felt a little confusing to me.  Maybe something more like "(or a [backup payment method](#))" would be better?

Reply:
That sounds good. I was trying to keep the wording short so it wouldn't wrap too much, but I think this works ok.

Reply:
Yeah, I think it looks better.  Speaking of wrapping, it seems there's a `display: block`on the link, which makes it wrap in a slightly confusing way?  I'm not sure which is better.

Current behavior (at desktop widths):

![current-behavior](https://user-images.githubusercontent.com/235183/142648010-e0cf5fd5-c497-4be3-99c0-01e6c661d984.png)

With `display: block` removed:

![display-block-removed](https://user-images.githubusercontent.com/235183/142648012-731aeab8-5c5f-431a-bfe5-96669b971a8a.png)

Another option could be to embrace the wrapping and just use slightly longer text, something like this?

![new-wording](https://user-images.githubusercontent.com/235183/142649426-4ef5c254-5605-4599-bc35-22dd6e97742f.png)


I think it might be OK since the section next to it is long also, and the more descriptive text might explain the situation better too.

Reply:
Nice find. That break bothered me but I didn't realize it was because there was a block display. I've removed it now, I think.

I'm going to opt for keeping the short text in both places for now, because it will be one less thing to localize, but I'll keep it in mind depending on how this turns out.